astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -965,7 +966,7 @@ def test_masked_vals_in_array_subtypes():
     assert t2.colnames == t.colnames
     for name in t2.colnames:
         assert t2[name].dtype == t[name].dtype
-        assert type(t2[name]) is type(t[name])  # noqa
+        assert_type_is_type(t2[name], t[name])
```

Comment:
A bit sad to have to introduce a new function just to avoid code checkers!  I'd prefer the relevant `noqa`...

Reply:
I undid this change in 757d1d9fd8c8b55ade21f6dc2d566c9ca5e8047f